tests/resources/ray/ray-simple-test.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ray-simple-test
+  labels:
+    app: ray-simple-test
+spec:
+  replicas: 1
+  selector:
+    type: Recreate
+    matchLabels:
+      app: ray-simple-test
+  template:
+    metadata:
+      labels:
+        app: ray-simple-test
+    spec:
+      containers:
+      - name: ray-odh-tests
+        image: quay.io/michaelclifford/ray-odh-tests:latest
```

Comment:
Ideally we'd have this image in the ODH org. Where's the dockerfile for this image defined?

Reply:
Dockerfile is currently here: https://github.com/redhat-et/ray-on-the-edge-demo/tree/main/tests 

Should these files be included in this repo under something like `ray/tests/`? 

What it the process to get images into the ODH org?